cmd/squid-store-id/main.go

```diff
@@ -31,13 +23,10 @@ func isChannelID(s string) bool {
 	return err == nil && val >= 0
 }
 
-// normalizeStoreID normalizes the store-id for caching by removing query parameters from CDN and S3 URLs.
+// normalizeStoreID normalizes the store-id for caching by removing query parameters from CDN URLs.
+// Squid's store_id_access ACL ensures this helper only receives CDN URLs (see squid.conf).
```

Comment:
nit: there are multiple at the moment

```suggestion
// Squid's store_id_access ACLs ensure this helper only receives CDN URLs (see squid.conf).
```

Reply:
updated